ui/admin/setup-edit.php

```diff
@@ -168,7 +168,7 @@
                     ),
                     'class' => 'pods-validate pods-validate-required'
                 ) ); ?>
-                    <input type="button" class="save-button button" value="OK" /> <a class="cancel" href="#cancel-edit">Cancel</a>
+                    <input type="button" class="save-button button" value="<?php _e( 'OK', 'pods' ); ?>" /> <a class="cancel" href="#cancel-edit"><?php _e( 'Cancel', 'pods' ); ?></a>
```

Comment:
@jamesgol this will break the translation of the "OK" button and include a static "OK" in front of the button.
The "OK" button text is used exactly from `value`.
Have you tested the code you've posted above?